lib/oli_web/live/delivery/student/prologue_live.ex

```diff
@@ -205,6 +206,12 @@ defmodule OliWeb.Delivery.Student.PrologueLive do
   attr :request_path, :string
 
   defp attempt_summary(assigns) do
+    activity_attempts =
+      Repo.preload(assigns.attempt, activity_attempts: [:part_attempts]).activity_attempts
```

Comment:
This will make a database call for every instance of attempt summary rendered. We should instead be
preloading the part attempt data up front with whatever database call is being made for the
attempt or perhaps with an async call during connected mount

Reply:
It makes sense. I'll fix it.